.buildconfig-android.yml

```diff
@@ -109,11 +109,3 @@ projects:
     - name: full-megazord-forUnitTests
       type: jar
     description: Megazord containing all features
-  # See https://github.com/mozilla/application-services/issues/5341
```

Comment:
Did you delete this, or was it the script? @jeddai re-adds it in #5348. Just so that Charlie is aware that they have to re-add it when they get the merge conflict

Reply:
The script did that.  I'm thinking we should undo this for the release, does that make sense to you?

Reply:
Yeah sounds good! 🚢